libs/core/src/lib/carousel/carousel.module.ts

```diff
@@ -0,0 +1,11 @@
+import { NgModule } from '@angular/core';
+import { CommonModule } from '@angular/common';
+import { CarouselComponent } from './carousel.component';
+import { CarouselItemComponent } from './carousel-item/carousel-item.component';
+
+@NgModule({
+    declarations: [CarouselComponent, CarouselItemComponent],
+    imports: [CommonModule],
+    exports: [CarouselComponent, CarouselItemComponent]
+})
+export class CarouselNewModule {}
```

Comment:
Only core we have a carousel why should we have "New"

Reply:
using CarouselModule now.